lib/browser/api/web-contents.js

```diff
@@ -183,22 +183,25 @@ for (const method of webFrameMethods) {
   }
 }
 
-const executeJavaScript = (contents, code, hasUserGesture) => {
-  return ipcMainUtils.invokeInWebContents(contents, false, 'ELECTRON_INTERNAL_RENDERER_WEB_FRAME_METHOD', 'executeJavaScript', code, hasUserGesture)
+const waitTillCanExecuteJavaScript = async (webContents) => {
```

Comment:
```suggestion
const waitTillCanExecuteJavaScript = (webContents) => {
```

Reply:
We can't do that because of the next line doing a short-circuit `return`.  Putting `return Promise.resolve()` just seems weird when putting `async` here works fine